frontend/src/api/resource.ts

```diff
@@ -0,0 +1,459 @@
+/* eslint-disable */
+import { util, configure, Reader, Writer } from "protobufjs/minimal";
```

Comment:
Here we generate protobufs for typescript. And as it has services it won't work in the web.
I think I tried to compile it before and it fails.

But as we use protobuf messages for websockets I kept that enabled for all `.proto` files.
There is option to add one more flag that will enable either gateway mode or protobuf mode for each `.proto` file, but I didn't like how it looked in the code, but can bring that back.